images/hub/jupyterhub_config.py

```diff
@@ -47,6 +47,11 @@ def get_secret(key, default=None):
 # Max number of servers that can be spawning at any one time
 c.JupyterHub.concurrent_spawn_limit = get_config('hub.concurrent-spawn-limit')
 
+active_server_limit = get_config('hub.active-server-limit', None)
+
+if active_server_limit:
```

Comment:
probably best to check `is not None` to catch the valid case of `active_server_limit = 0` for no limit.

Reply:
Good catch! Done!